rdr_service/offline/tactis_bq_sync.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import string
+from typing import Union
+
+from datetime import datetime
+
+import google.cloud.bigquery
+import google.api_core.exceptions
+from google.cloud import bigquery
+from google.cloud.exceptions import NotFound
+
+_logger = logging.getLogger("rdr_logger")
+
+
+class TactisBQDataSync:
+
+    EXTERNAL_CONNECTION = {
+        "all-of-us-rdr-sandbox": "all-of-us-rdr-sandbox.us-central1.aou_sandbox_rdrmaindb",
+        "all-of-us-rdr-stable": "all-of-us-rdr-stable.us-central1.all-of-us-rdr-stable-mysql",
+        "all-of-us-rdr-prod": "all-of-us-rdr-prod.us-central1.bq-rdr-preprod-curation"
+    }
+
+    QUERIES = {
+        "modified_participants": """
+                        SELECT * FROM
+                          EXTERNAL_QUERY("{external_connection}",
+                            "SELECT t.participant_id, ps.first_name, ps.middle_name, ps.last_name, ps.email, """
+                                 """ps.phone_number, ps.participant_origin FROM participant_data_to_tactis t """
+                                 """LEFT JOIN participant_summary ps ON ps.participant_id = t.participant_id """
+                                 """LEFT JOIN participant p ON p.participant_id = ps.participant_id """
+                                 """WHERE p.is_test_participant = 0 AND p.is_ghost_id is null """
+                                 """AND p.hpo_id not in (21) AND t.created >= '{since_date}'");"""
+    }
```

Comment:
We try to mainly lean into the ORM for SQL queries,  so I think it would be best to adhere to that and create a DAO class for this and have this query in that. So that way you can initialize the dao and pull the designated query from that.